node/pkg/governor/governor_monitoring.go

```diff
@@ -96,14 +95,16 @@ import (
 )
 
 // Admin command to display status to the log.
-func (gov *ChainGovernor) Status() string {
+func (gov *ChainGovernor) Status() (resp string, err error) {
```

Comment:
I changed the function signature here because `sumValue` can theoretically overflow though in practice should never do so.

Two approaches here:
1. We can swallow the error and just return 0 and log
2. I can keep pulling at this thread and propagate the error through the codebase. It might make this PR needlessly complex for such an edge-case.

Let me know what seems right.

Reply:
Yeah I think my preference is to not return the error. Thanks I'll issue a commit soon to change this.